packages/@okta/vuepress-site/docs/guides/set-up-self-service-registration/main/index.md

```diff
@@ -0,0 +1,175 @@
+
+    * Go to **Directory** and then **Profile Editor**.
+    * Click **Profile** for the **Okta User (default)** profile.
+    * Find the attribute that you added, and then click its information icon.
+    * Change the **User permission** drop-down box to **Read - Write**.
+    * Click **Save Attribute**.
+
+## Disable the security image and additional self-service recovery options
+
+With Okta, new users are prompted to choose a security image and optionally an additional security question prompt when they first sign in. To provide your users with a seamless and smooth sign-in experience, you may need to disable these options.
+
+1. In the Admin Console, go to **Settings** > **Customization**.
+
+2. In the **Optional User Account Fields** section, click **Edit**.
+
+3. Select **Disabled** from the **Security image** drop-down box, and then click **Save**.
+
+If you don't have any policies configured or your org doesn't allow you to disable additional self-service recovery options, you can skip the following steps. Disabling additional self-service recovery options is an Early Access feature. To enable it, contact [Okta Support](https://support.okta.com/help/s/?_ga=2.17747641.1660906902.1597076228-1076744453.1575496867).
+
+1. Go to **Security** > **Authentication**.
+
+2. Select a policy from the left, click **Edit**, and scroll down to **ACCOUNT RECOVERY**.
+
+3. Clear the **Security question** check box in the **Additional self-service recovery** section.
+
+4. Click **Update Policy**.
+
+5. Repeat steps 3 to 4 for any other policies that you have configured.
+
+## Configure registration in the Okta Sign-in Widget
+
+### Configure registration in a custom sign-in page in the Okta-hosted Widget
+
+If you left the [**Add to Sign-In Widget**](#enable-and-configure-a-self-service-registration-policy) check box clear when you configured the registration policy, then you need to configure a link using JavaScript in the **Customize Sign-In Page** HTML editor.
+
+> **Note:** To enable the Custom Sign-In Page HTML editor, you must have a [custom URL domain](/docs/guides/custom-url-domain/overview/) configured.
+
+1. In the Admin Console, go to **Settings** > **Customization**.
+2. Click the **Custom Sign In** tab.
+3. In the HTML editor, add the following configuration parameters directly under the `var config = OktaUtil.getSignInWidgetConfig();` line:
+
+```javascript
+config['features.registration'] = true;
+config['authScheme'] = 'SESSION';
+```
+
+4. Click **Save and Publish**.
+
+### Configure self-service registration in an embedded or self-hosted Widget
+
+Before you can configure self-service registration in your embedded or self-hosted Widget, ensure that you first enable [self-service registration](#enable-and-configure-a-self-service-registration-policy). Then you can set `feature.registration` in the Widget to `true`.
+
+```json
```

Comment:
we don't typically put notes in code examples...can you pull the note out and put it above the example?